fdbserver/Status.actor.cpp

```diff
@@ -1241,6 +1241,11 @@ ACTOR static Future<JsonBuilderObject> dataStatusFetcher(WorkerDetails ddWorker,
 				stateSectionObj["name"] = "healthy_removing_server";
 				stateSectionObj["description"] = "Removing storage server";
 			}
+			else if (highestPriority >= PRIORITY_TEAM_HEALTHY) {
```

Comment:
I'm thinking what is the state's priority between `PRIORITY_REBALANCE_SHARD` and `PRIORITY_TEAM_CONTAINS_UNDESIRED_SERVER `. 
Since the lower priority it is, the better the cluster situation is.
Why this situation is worse than `PRIORITY_REBALANCE_SHARD `?

Reply:
Not sure why PRIORITY_TEAM_HEALTHY is 140 and PRIORITY_REBALANCE_SHARD is 100.

Reply:
I see the logic now:
The larger the value is, the higher the priority is.
It is possible that we have all teams healthy but the data distribution needs to balance data across teams.